db/migrate/20200206212235_devise_token_auth_create_users.rb

```diff
@@ -0,0 +1,13 @@
+class DeviseTokenAuthCreateUsers < ActiveRecord::Migration[5.2]
```

Comment:
When it was generated, this created a whole user, so I stripped it down a lot, based on a guide. I'm not sure what of these are actually required, so I'll probably be able to cut this down a lot more.